src/main/java/com/nina/modelling/shoes/KidShoe.java

```diff
@@ -11,21 +11,24 @@ public KidShoe(String category, String accessory, String type) {
         this.type = type;
     }
 
-    public KidShoe(String category){
+    public KidShoe(String category) {
         this.category = category;
     }
-    public KidShoe(String accessory, String type){
+
+    public KidShoe(String accessory, String type) {
         this.accessory = accessory;
         this.type = type;
     }
 
-    public String category(){
+    public String category() {
         return category;
     }
-    public String getAccessory(){
+
+    public String Accessory() {
         return accessory;
     }
-    public String getType(){
+
+    public String Type() {
```

Comment:
If you changed these getters to naming without get, make this change for all getters in the code.